src/main/scala-2.11/markets/package.scala

```diff
@@ -13,4 +13,34 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-package object markets
+import akka.actor.ActorRef
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1425046398/issues/source?bid=2921362&fileBranchId=2987952#l16)